src/cargo/util/command_prelude.rs

```diff
@@ -373,12 +373,7 @@ pub trait ArgMatchesExt {
                 for timing_output in timing_output.split(',') {
                     let timing_output = timing_output.to_ascii_lowercase();
                     let timing_output = match timing_output.as_str() {
-                        "html" => {
-                            config
-                                .cli_unstable()
-                                .fail_if_stable_opt("--timings=html", 7405)?;
-                            TimingOutput::Html
-                        }
```

Comment:
I don't think this should have changed?  The CLI options should remain unstable.

Reply:
Should html config also be unstable?

Reply:
AFAIK, specifying the format of timing report in CLI option is unstable regardless of the format specified.

https://doc.rust-lang.org/1.60/cargo/reference/unstable.html#timings

Reply:
It is also confusing that the html options document is not marked as unstable. And I have a question, why is -timings stable, but -timings=html is not?

Reply:
[Here is the original discussion](https://github.com/rust-lang/cargo/pull/10245#discussion_r787103783)

Reply:
Got it. Thanks!

Reply:
Addressed! Now they're all unstable.

Reply:
Also I marked the html instability in the documentation to reduce confusion 😖.

Reply:
Sorry for the confusion @hi-rustin, but this still doesn't appear correct.  The only change I requested is to "make sure the JSON option isn't stabilized.".   We avoided adding the ability to specify the format in the CLI, but I think to keep things simple in the config, it should allow:

* `build.timings = ["html"]`: This is stable.
* `build.timings = ["json"]`: This is unstable.

The JSON format should be unstable, the HTML one is not.

Reply:
How about now? check out this commit https://github.com/rust-lang/cargo/pull/10567/commits/d6b6f8d8c6f2d8fed61426a704602bda5dcf259d. 
Thanks for your review!  💚 💙 💜 💛 ❤️